### DISCUSSION TITLE
不同模式的包，如何进行选择 ？

### DISCUSSION THREAD

**Discussion by q962** [xmake-io/xmake#4185] [Q&A] [answered]
不同模式的包，如何进行选择 ？

**Comment by waruqi** [comment, marked as answer]
xmake-repo 仓库的包，仅支持 release/debug 切换。。`add_requires("xxx", {debug = true})`

自己的包，可以通过 package/add_configs 和 `add_requires("xxx", {configs = {mymode = true}})` 自己处理模式配置
